gallery/visualtests/nxDropdown.js

```diff
@@ -86,6 +87,16 @@ describe('NxDropdown', function() {
 
       await checkScreenshot(targetElement, 251, 218);
     });
+
+    it('looks right when focused via keyboard', async function() {
+      const [targetElement] = await waitAndGetElements(selector);
+      const page = getPage();
+
+      await moveMouseAway();
+      await page.keyboard.press('ArrowLeft');
```

Comment:
I don't see any behavior for the Left arrow key defined in the ARIA docs; what's this for?